### ISSUE TITLE
Falling off of the chasm of the Medisim while on the Icebox map will spawn you on the lower z-level

### ISSUE THREAD

**Issue by TheSilverNuke** [tgstation/tgstation#73960]
Reporting client version: 514.1589

## Round ID:
201724

## Testmerges:
- [Adds TTS to announcements made by the Captain and silicons](https://www.github.com/tgstation/tgstation/pull/72300)
- [Nanotrasen Budget Programme - Mothball Edition [BIRDSHOT STATION]](https://www.github.com/tgstation/tgstation/pull/73502)
- [Don't cache \ref on 515](https://www.github.com/tgstation/tgstation/pull/73902)

## Reproduction:

Basically spawn on the medisim shuttle while the icebox map is active, fall into a chasm, then you're free to go to the station! You'll encounter an issue with being unable to breathe but that's small in comparison to infinitely returning to the station as a ghost.

Specifically I was carrying a flag and got shoved by an opponent into the chasm and they tumbled along with me

**Comment by Sealed101** [comment]
duplicate of #56695
